scripts/check-bip-dns.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+# zenoss-inspector-info
+# zenoss-inspector-tags docker
+# zenoss-inspector-deps docker-config.sh
+
+import subprocess as sp
+
+def main():
+    cmd = ["ip", "addr", "show", "docker0"]
+    p = sp.Popen(cmd, stdout=sp.PIPE, stderr=sp.PIPE)
+    stdout, stderr = p.communicate()
+    if p.returncode != 0:
+        raise sp.CalledProcessError(p.returncode, ' '.join(cmd))
+    bip = "--bip=" + stdout.split('\n')[2].strip().split()[1]
```

Comment:
The following is a readability (clean code) nit-pick intended to make it a little more clear what we're parsing out of the command output
```
ipV4InfoLine =  stdout.split('\n')[2].strip()
ipv4AddrWithNetmask = ipV4InfoLine.split()[1]
ipv4AddrWithoutNetmask = ipv4AddrWithNetmask.split()[1]
bip = "--bip=" + ipv4AddrWithNetmask
dns = "--dns=" + ipv4AddrWithoutNetmask
```